Lib/gftools/packager.py

```diff
@@ -914,7 +920,12 @@ def _create_or_update_metadata_pb(upstream_conf: YAML,
   for font in metadata.fonts:
     font.name = upstream_conf['name']
   metadata.designer = upstream_conf['designer']
-  metadata.category = upstream_conf['category']
+
+  while metadata.category:
+    metadata.category.pop()
```

Comment:
Would be nice if there was a method to clear this container but it doesn't [exist](https://googleapis.dev/python/protobuf/latest/google/protobuf/internal/containers.html#google.protobuf.internal.containers.RepeatedScalarFieldContainer) hence popping till empty.

Reply:
All of the following work ([via](https://developers.google.com/protocol-buffers/docs/reference/python-generated#fields)):

```py
metadata.category[:] = []
metadata.ClearField('category')
del metadata.category[:]
```

Reply:
Thanks. I've opted for ClearField.